sdk/tests/conformance/reading/00_test_list.txt

```diff
@@ -1,3 +1,3 @@
 read-pixels-pack-alignment.html
 read-pixels-test.html
-
+fbo-remains-unchanged-after-read-pixels.html
```

Comment:
All new tests should have the min version condition so that browsers passing older version of conformance won't regress; it's also for bookkeeping purpose so people know this is a newly added tests after 1.0.3 snapshot.